config.yml

```diff
@@ -14,14 +14,10 @@ model:
   input:
     location: 'data/schema_org_schemas/HTAN.jsonld'
     file_type: 'local'
+    validation_schema: 'data/validation_schemas/HTAN_validation_schema_FamilyHistory.json'
```

Comment:
@sujaypatil96 we can merge the PR with this schema, but in the next iteration let's have the example validation schema.